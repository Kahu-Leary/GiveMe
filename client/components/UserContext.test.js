import React from 'react'
import renderer from 'react-test-renderer'

import { UserProvider } from './UserContext'

it('renders correctly', () => {
  const tree = renderer
    .create(

      <UserProvider>
        <div>
              this is a child
        </div>
      </UserProvider>

    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
