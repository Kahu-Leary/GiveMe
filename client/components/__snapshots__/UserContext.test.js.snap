// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  this is a child
</div>
`;
